docs/custom_theme/home.html

```diff
@@ -7,10 +7,10 @@
         <div class="mdx-hero__content">
           <h1>Welcome to the {{ config.site_name }}</h1>
           <p>{{ config.site_description }}</p>
-          <a href="{{ 'get-started/quick-start-guide' | url }}" title="Get Started" class="md-button md-button--primary md-button--teriary">
+          <a href="/{{ 'get-started/quick-start-guide' | url }}" title="Get Started" class="md-button md-button--primary md-button--teriary">
             Get started
           </a>
-          <a href="{{ 'get-started/get-your-planet-account' | url }}" title="Get your Planet account" class="md-button md-button--secondary">
+          <a href="/{{ 'get-started/get-your-planet-account' | url }}" title="Get your Planet account" class="md-button md-button--secondary">
```

Comment:
```suggestion
          <a href="./{{ 'get-started/get-your-planet-account' | url }}" title="Get your Planet account" class="md-button md-button--secondary">
```

Reply:
Thanks for reviewing and testing JuLee, I'll incorporate the changes.